packages/web-components/README.md

```diff
@@ -14,17 +14,96 @@ A Carbon Design System variant that's as easy to use as native HTML elements, wi
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Getting started with development
+## Getting started
 
-1. Fork this repository and clone it
-2. `yarn install`
-3. `yarn wca && yarn storybook`
+To install `@carbon/ibmdotcom-web-components` in your project, you will need to run the
+following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S @carbon/ibmdotcom-web-components lit-html lit-element
+```
+
+If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
+instead:
+
+```bash
+yarn add @carbon/ibmdotcom-web-components lit-html lit-element
+```
+
+### Basic usage
+
+The first thing you need is **setting up a module bundler** to resolve ECMAScript `import`s. Once you set up a module bundler, you can start importing our component modules, like:
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/link-with-icon/link-with-icon';
+```
+
+Once you do that, you can use our components in the same manner as native HTML tags, like:
+
+```html
+<dds-link-with-icon href="http://ibmdotcom-web-components-canary.mybluemix.net/">
+  Link text
+  <svg
+    slot="icon"
+    focusable="false"
+    preserveAspectRatio="xMidYMid meet"
+    xmlns="http://www.w3.org/2000/svg"
+    aria-hidden="true"
+    width="20"
+    height="20"
+    viewBox="0 0 20 20"
+  >
+    <path d="M11.8 2.8L10.8 3.8 16.2 9.3 1 9.3 1 10.7 16.2 10.7 10.8 16.2 11.8 17.2 19 10z"></path>
+  </svg>
+</dds-link-with-icon>
+```
 
 ## Browser support
 
 - Latest Chrome/Safari/FF ESR
 - IE and classic Edge are _not_ supported
 
+## List of available components
+
+View available web components at: http://ibmdotcom-web-components-canary.mybluemix.net/. You can see usage information in several ways:
+
+1. Going to Docs tab, where it shows the usage and available attributes, properties and custom events.
+2. Clicking the **KNOBS** tab at the bottom and changing values there. Most knobs are shown as something like `Button kind (kind)`, where `kind` is the attribute name
+3. Clicking the **ACTION LOGGER** tab at the bottom and interacting with the selected component. You may see something like `bx-modal-closed` which typically indicates that an event with such event type is fired.
+
+## Web Components way for analytics
+
+`@carbon/ibmdotcom-react`, etc. libraries have `data-autoid` attributes in several places, to convey better context than raw HTML `<div>`, etc. tags and their attributes when analytics libraries captures bubbling events e.g. `click` on `<body>` and inspects `event.target`.
```

Comment:
At minimum, should we add the data-autoid for the parent level tags? e.g. `data-autoid="dds--masthead"`?

Reply:
We could do `<dds-masthead data-autoid="dds--masthead">`. That said, `<dds-masthead>` tag name seems to serve the purpose of `data-autoid`, but did you meant that we'd want to duplicate it...?

Reply:
I'm wondering if for analytics they are tracking anything across different applications within IBM.com, and if they need to have that consistent stable selector language when going from one part of the site to another? I might be overthinking it though.

Reply:
Yeah the thought overall makes sense. Given `data-autoid` inside shadow DOM doesn't make `event.target.dataset.audoId` work due to [event retargeting](https://developers.google.com/web/fundamentals/web-components/shadowdom#events) as discussed, I thought that providing a new set of guidelines makes more sense instead of providing `data-autoid` that makes it looks like "compatible" but not actually fully.

But if we want to go with "partial compatibility" route I'm all for it.